assets/css/tinyfilemanager.css

```diff
@@ -68,10 +68,18 @@ textarea {
 }
 
 .file-tree-view+#main-table {
-/*  width: 75%!important;
+  /*  width: 75%!important;
```

Comment:
am besten macht man solche dinge, die eigentlich nix mit einander zu tun haben, in separaten PRs.
(also die CSS Anpassungen vs. die PHP Session Anpassungen).

Aber wird schon gehen - insofern du es getestet hast ;)

Reply:
Passt ja ;) Und da hast du wohl Recht. Hätte ich in diesem Fall wohl nochmal einen eigenen Branch anlegen müssen, so ging es nun direkt hier in den PR mit rein.

Reply:
Jep genauso. Aber ich denke es wird so gehen